src/app/component/comments/components/likes-counter/likes-counter.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { CommentsService } from '../../services/comments.service';
+
+@Component({
+  selector: 'app-likes-counter',
+  templateUrl: './likes-counter.component.html',
+  styleUrls: ['./likes-counter.component.scss']
+})
+export class LikesCounterComponent implements OnInit {
+  @Input() commentId: number;
+  public likeImg = 'assets/img/comments/like.png';
+  public likes: number;
+
+  constructor(private commentsService: CommentsService) { }
+
+  ngOnInit() {
+    this.getLikesFromServer();
+  }
+
+  private getLikesFromServer(): void {
+    this.commentsService.getCommentLikes(this.commentId)
+      .subscribe(this.setLikes.bind(this));
```

Comment:
Please change this method to not use bind();

Reply:
> Please change this method to not use bind();

Won't work without bind

Reply:
Everything work as before
![image](https://user-images.githubusercontent.com/56609118/86582637-9b4b0b00-bf8a-11ea-99da-af767b85572a.png)
![image](https://user-images.githubusercontent.com/56609118/86582665-a7cf6380-bf8a-11ea-8a46-7262f7fd77bf.png)

Reply:
> Everything work as before
> ![image](https://user-images.githubusercontent.com/56609118/86582637-9b4b0b00-bf8a-11ea-99da-af767b85572a.png)
> ![image](https://user-images.githubusercontent.com/56609118/86582665-a7cf6380-bf8a-11ea-8a46-7262f7fd77bf.png)

Well, after adding types something may not work, that is why the binding is used here
![works](https://user-images.githubusercontent.com/60985783/86784281-fdc70880-c069-11ea-92a6-d9d82fc8f9e0.png)

Reply:
Please add types.